extension/src/content/index.js

```diff
@@ -482,4 +486,4 @@ function init() {
   }
 }
 
-init();
+chrome.storage.sync.get(init);
```

Comment:
This isn't necessarily a blocker (we can fix this later if need be), but this only runs when the page is loaded, correct? One use case this might not handle very gracefully is when a GitHub/StackOverflow tab is open and someone goes and changes the settings (they wouldn't go into effect until the page is reloaded).

We could probably fix this by reading from chrome storage each time Prettier formats (again, does not need to be a blocker to land this, just want to make sure it's commented here for posterity).

Reply:
I forgot to implement this, but we should be able to use [`chrome.storage.onChanged.addListener `](https://developer.chrome.com/extensions/storage#event-onChanged)